modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc

```diff
@@ -9,5 +9,28 @@ include::project-docs:partial$attributes.adoc[]
 {description}
 
 
-Distributed ACID Transactions are not currently available for the Scala SDK.
-include::{version-common}@sdk:shared:partial$acid-transactions.adoc[tag=other]
+Distributed ACID Transactions are implemented in the Java SDK, and simple to integrate into your Scala client code.
+
+
+== Java Transactions from the Scala SDK
```

Comment:
Yes, I think we could say "from the Scala or Kotlin SDK", and "in your Scala or Kotiln app" down below.

Might also be worth mentioning the user needs to include the Couchbase Java SDK in their project.

Reply:
Sorry, I missed that this page is specifically for `docs-sdk-scala`. s/Scala/Kotlin would work for `docs-sdk-kotlin`.